backend/src/helpers/new-cars/new-cars-helper.ts.ts

```diff
@@ -0,0 +1,14 @@
+import { getNewCar } from '@services/new-cars/new-cars.service';
+
+import type { ModelResponseDto } from '@autoline/shared/common/types/types';
+
+const newerCars = async (quantity: number): Promise<ModelResponseDto[]> => {
```

Comment:
1. Function name, described above.
2. `quantity` is a too complex word for this function and the use case. It can be named as `amount`